src/subtitles-octopus.js

```diff
@@ -1,4 +1,16 @@
 var SubtitlesOctopus = function (options) {
+    var supportsWebAssembly = false;
```

Comment:
Maybe extract that into a separate function?

Reply:
I wanted to _not_ pollute the global namespace, thus all things are contained within this function.